lms/djangoapps/verify_student/management/commands/approve_id_verifications.py

```diff
@@ -8,7 +8,7 @@
 import time
 from pprint import pformat
 
-from django.contrib.auth.models import User  # lint-amnesty, pylint: disable=imported-auth-user
+from django.contrib.auth.models import User  # lint-amnesty, pylint: disable=imported-auth-user, unused-import
```

Comment:
If this import isn't used, could we remove it instead of ignoring the lint error?

Reply:
Bump.